src/common/dao/replication_job.go

```diff
@@ -136,6 +137,114 @@ func AddRepPolicy(policy models.RepPolicy) (int64, error) {
 	return policyID, nil
 }
 
+// AddRepPolicyNg ...
+func AddRepPolicyNg(policy models.RepPolicyNg) (int64, error) {
```

Comment:
We'll remove all `ng` at last, so can we rename the function name? We can create a new folder called `ng` and put the functions in the folder.

Reply:
OK